dspace/config/spring/api/task-scheduler.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:task="http://www.springframework.org/schema/task"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+    http://www.springframework.org/schema/beans/spring-beans.xsd
+    http://www.springframework.org/schema/task
+    http://www.springframework.org/task/spring-task.xsd">
+
+  <!-- Configure the scheduler -->
+  <task:scheduler id="myScheduler" pool-size="10"/>
```

Comment:
can we found a way to make easier turn off the scheduler without touching the xml file? an option could be our own implementation wrapping the default one with an additional boolean flag, if turned off the scheduler will just silent.
This is needed in deployment where there are multiple nodes, off course we must have the scheduler running only on a node (where we can set the property to true using ENV variable)

Reply:
That seems to be quite some overhead to avoid changing the config
Removing the scheduler would achieve the same result

Reply:
it would be hard to change the xml file just on a single node than set an ENV variable, if this is too complex to do in this PR I would like at least to have a ticket to keep track of that. We should start a section in the documentation to explain how to deploy dspace in cluster and point to this and other features that would require the selection of a "main" node

Reply:
For what it's worth, it from this answer it *sounds like* setting the `cron = -` will disable the schedule: https://stackoverflow.com/a/57655646

So, this is even more argument for moving the `cron` to our Configuration files...as that'd let you easily change it or enable/disable the task via your `local.cfg`, or environment variables (as our ConfigurationService also reads env)

Reply:
It's also possible to enable/disable by configurations + annotations like this: https://stackoverflow.com/a/51198831  That also should be possible in DSpace, as Spring annotations can read from our configuration.  For example, we already use `@ConditionalOnProperty` to do things like enable/disable OAI-PMH via an `oai.enabled` configuration: https://github.com/DSpace/DSpace/blob/main/dspace-oai/src/main/java/org/dspace/xoai/controller/DSpaceOAIDataProvider.java#L56

Reply:
both options look fine to me (thanks @tdonohue for the investigation). I slightly prefer the last ConditionalOnProperty over the first but both are fine

Reply:
@tdonohue Marie has verified the options in detail.
The `cron = -` syntax is not supported (it's not in the JavaDocs, and testing it results in `Cron expression must consist of 6 fields (found 1 in "-")`
The `@ConditionalOnProperty` is also not supported, most likely it can only be used if everything is moved to Java.

I don't see any options to configure this in local.cfg without bigger Java changes, which may be out-of-scope if it only has the goal to ease overriding config